resources/stylesheets/page/_login.scss

```diff
@@ -0,0 +1,108 @@
+body {
+    background-image: $back_img;
+    background-size: cover;
+    background-color: $theme_bl;
+    font-family: monospace;
+    @include flex-center;
+    width: 100vw;
+    height: 100vh;
+}
```

Comment:
this seems major change to the body element, the main.scss will be apply to all the project, if this dont cause any error thru all the app, then move this to the _base.scss since the is use the same body for all the project

Reply:
Done, it shouldn´t cause any error.